spec/lib/kube_link_generator_spec.rb

```diff
@@ -167,7 +167,8 @@ def build_answers
       it 'should return the expected information' do
         role = 'dummy-role'
         job = 'dummy'
-        pods = specs._get_pods_for_role(job)
+        sts = "docker.io/image-one\ndocker.io/image-two"
```

Comment:
I think this (and others in this file) should be `sts_image` instead of `sts`.

Reply:
will update

Reply:
Done with commit 9f5818a58bc40cb911cbd4b8f71ed8f481cf6e1c